EIPS/eip-draft_hfmeta_spuriousdragon.md

```diff
@@ -0,0 +1,30 @@
+## Preamble
+
+    EIP: <to be assigned>
+    Title: Hardfork Meta: Spurious Dragon
+    Author: Alex Beregszaszi
+    Type: Standard Track
+    Category: Core
+    Status: Final
+    Created: 23/04/2017
+    Requires: 155, 160, 161, 170
+
+## Abstract
+
+This specifies the changes included in the hard fork named Spurious Dragon.
+
+## Specification
+
+- Codename: Spurious Dragon
+- Activation:
+  - Block >= 2,675,000 on Mainnet
```

Comment:
I would find it extremely useful if these meta EIPs get merged. Always searching my way through the pull requests when looking for an overview of fork EIPs.